src/SDPoptimize.jl

```diff
@@ -182,9 +187,9 @@ Returns :
     of the system at each time step
 
 """
-function sdp_optimize(model::SPModel,
+function solve_DP(model::SPModel,
```

Comment:
Personally I prefer solve_SDP because SDDP solve also dynamic programming equations. But if the others agree, I'm OK